src/intro.adoc

```diff
@@ -3,5 +3,5 @@
 The RISC-V IO Mapping Table (RIMT)  provides information about the RISC-V IOMMU
 cite:[IOMMU-SPEC] and the relationship between the IO topology and the IOMMU in
 ACPI cite:[ACPI-SPEC] based RISC-V platforms. The RIMT identifies which
-components are behind IOMMU and how they are connected together. RISC-V IOMMU
+components are backing IOMMU and how they are connected together. RISC-V IOMMU
```

Comment:
"behind" is more directional to me so I suggest "backing". Please let me know if you want to discuss this farther. Maybe there is a third word that is better?